lib/components/map/enhanced-stop-marker.js

```diff
@@ -101,6 +101,8 @@ class EnhancedStopMarker extends Component {
     const { activeStopId, highlight, languageConfig, modeColors, stop } =
       this.props
     const { code, id, lat, lon, name } = stop
+    if (!id) return null
+
     const [, stopId] = id.split(':')
     const stopCode = code || stopId
     if (!stopCode) return null
```

Comment:
Can line 104 & 108 be consolidated?

Reply:
If only! `stopCode` relies on `id` exisitng